src/simularium/versionHandlers.ts

```diff
@@ -1,27 +1,94 @@
+import { mapValues } from "lodash";
 import * as si from "si-prefix";
 
 import { DEFAULT_CAMERA_SPEC } from "../constants";
 import {
+    CubeDisplayType,
+    GizmoDisplayType,
+    ObjDisplayType,
+    PdbDisplayType,
+    SphereDisplayType,
     TrajectoryFileInfo,
     TrajectoryFileInfoAny,
     TrajectoryFileInfoV1,
 } from "./types";
+import { AgentDisplayDataWithGeometry } from "./VisGeometry";
```

Comment:
this file shouldn't depend on VisGeometry, i'd define it here and import it the other way

Reply:
or in `types`